src/utils/probe-utils.js

```diff
@@ -55,19 +45,20 @@ function createNewError(which) {
   return error;
 }
 
-export const prepareForQuantilePlot = (probeData, key = 'version') =>
+export const transformQuantileResponse = (probeData, key = 'version') =>
   probeData.map((probe) => {
-    const h = probe.data[0].histogram;
+    const h = probe.histogram;
     if (!h) {
       throw createNewError('MISSING_HISTOGRAM');
     }
     const histogram = responseHistogramToGraphicFormat(h);
-    const { percentiles } = probe.data[0];
+    const { percentiles } = probe;
 
     if (!percentiles) {
       throw createNewError('MISSING_PERCENTILES');
     }
 
+    // fixme: remove need for transformedPercentiles.
```

Comment:
Can we change this to `FIXME`? I tend to assume capitalization when searching for these keywords.